openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -54,3 +75,288 @@ def send_grade_to_credentials(self, username, course_run_key, verified, letter_g
             f"Failed to send grade to credentials. Reason: {error_msg}"
         )
         raise self.retry(exc=exception, countdown=countdown, max_retries=MAX_RETRIES)
+
+
+@shared_task(base=LoggedTask, ignore_result=True)
```

Comment:
This `LoggedTask` base means that this will log any time it fails. We can set up Splunk alerting on the stack trace. [PR about failure logging here.](https://github.com/edx/edx-celeryutils/pull/14)

Sample output when forcing an error:
```
2021-04-23 20:39:16,254 INFO 17875 [openedx.core.djangoapps.credentials.management.commands.notify_credentials] [user None] [ip None] notify_credentials.py:152 - notify_credentials starting, dry-run=False, site=None, delay=0 seconds, page_size=100, from=NA, to=NA, notify_programs=False, user_ids=None, execution=manual
2021-04-23 20:39:16,254 INFO 17875 [openedx.core.djangoapps.credentials.management.commands.notify_credentials] [user None] [ip None] notify_credentials.py:187 - 1 courses specified: course-v1:edX+101+3T2020
2021-04-23 20:39:16,319 ERROR 17875 [celery_utils.logged_task] [user None] [ip None] logged_task.py:44 - [bbc828db-765a-40df-85c0-e5e35617c36d] failed due to Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/celery/app/trace.py", line 412, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/edx_django_utils/monitoring/internal/code_owner/utils.py", line 179, in new_function
    return wrapped_function(*args, **kwargs)
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/credentials/tasks/v1/tasks.py", line 98, in handle_notify_credentials
    raise
RuntimeError: No active exception to reraise
2021-04-23 20:39:16,320 ERROR 17875 [celery.app.trace] [user None] [ip None] trace.py:255 - Task openedx.core.djangoapps.credentials.tasks.v1.tasks.handle_notify_credentials[bbc828db-765a-40df-85c0-e5e35617c36d] raised unexpected: RuntimeError('No active exception to reraise')
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/celery/app/trace.py", line 412, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/edx_django_utils/monitoring/internal/code_owner/utils.py", line 179, in new_function
    return wrapped_function(*args, **kwargs)
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/credentials/tasks/v1/tasks.py", line 98, in handle_notify_credentials
    raise
RuntimeError: No active exception to reraise
2021-04-23 20:39:16,320 INFO 17875 [celery_utils.logged_task] [user None] [ip None] logged_task.py:25 - Task openedx.core.djangoapps.credentials.tasks.v1.tasks.handle_notify_credentials[bbc828db-765a-40df-85c0-e5e35617c36d] submitted with arguments ({'verbosity': 1, 'settings': None, 'pythonpath': None, 'traceback': False, 'no_color': False, 'force_color': False, 'dry_run': False, 'site': None, 'courses': ['course-v1:edX+101+3T2020'], 'start_date': None, 'end_date': None, 'delay': 0, 'page_size': 100, 'auto': False, 'args_from_database': False, 'verbose': True, 'notify_programs': False, 'user_ids': None}, ['course-v1:edX+101+3T2020']), {}
```